project-koku/koku
# Feature/#1913 - Store ReportingCommonMap in static Python constant

This PR removes the ReportingCommonMap

- Generates Django migration to drop reportcolumnmap table. 

- Delete Report Column Map JSON files and replace their usage with a static Python constant.

- Change old migrations to stop inserting data into the report column map table from JSON. 

- Removes test that asserts the presence of the reportcolumnmap table.

## Testing Instructions:
Run the following commands:
Observe the koku-worker logs, on Mac 
```
docker-compose logs -f koku-worker
# (if someone knows the Fedora/RHEL similar command please comment with that)
```
Run the following:
```
make docker-reinitdb-with-sources
make load-test-customer-data
```

Assert that there are no processor errors in the koku-worker logs, except for being unable to find a manifest.

Assert that there is data in the following database tables:
- reporting_awscostentrylineitem_daily_summary

`SELECT * FROM acct10001.reporting_awscostentrylineitem_daily_summary ORDER BY id ASC LIMIT 100`

- reporting_azurecostentrylineitem_daily_summary

`SELECT * FROM acct10001.reporting_azurecostentrylineitem_daily_summary ORDER BY id ASC LIMIT 100`

- reporting_ocpusagelineitem_daily_summary

`SELECT * FROM acct10001.reporting_ocpusagelineitem_daily_summary ORDER BY id ASC LIMIT 100`
